odl/phantom/transmission.py

```diff
@@ -75,6 +75,11 @@ def _modified_shepp_logan_ellipses(ellipses):
     """
     intensities = [1.0, -0.8, -0.2, -0.2, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
 
+    # Add minimal numbers to ensure that the result is nowhere negative.
+    # This is needed due to numerical issues.
+    intensities[2] += 5e-17
+    intensities[3] += 5e-17
```

Comment:
This guy doesn't know about the data type and the numerical error seems to only appear for double precision. 

```python
>>> import odl
>>> space = odl.uniform_discr([0, 0], [1, 1], [100, 100], dtype='float32')
>>> np.all(np.greater_equal(odl.phantom.shepp_logan(space), 0))
```